docs/manager/graphql-reference/schema.graphql

```diff
@@ -1984,7 +1984,7 @@ type Mutations {
   clear_images(registry: String): ClearImages
 
   """Added in 25.4.0"""
-  purge_images(agent_id: String!, images: [ImageRefType]!): PurgeImages
+  purge_images(agent_id: String!, images: [ImageRefType]!, props: PurgeImageProps): PurgeImages
```

Comment:
QUERY: func(key KeyStruct, options Options (filter, order, query filter...))
---
CREATE: func(props CreateProps) pattern
UPDATE: func(key KeyStruct, updateProps UpdateProps) pattern
DELETE: func(key KeyStruct, options OptionProps) pattern
---
Now we need to fit it into the pattern above.